README.md

```diff
@@ -1,6 +1,6 @@
 <img src="docs/images/s2n_logo_github.png" alt="s2n">
 
-s2n-tls is a C99 implementation of the TLS/SSL protocols that is designed to be simple, small, fast, and with security as a priority. It is released and licensed under the Apache License 2.0. 
+s2n-tls is a C99 implementation of the TLS/SSL protocols that is designed to be simple, small, fast, and with security as a priority. It is released and licensed under the Apache License 2.0.
 
```

Comment:
You only made whitespace changes to this file-- why include it?

Reply:
Didnt mean to specifically make any changes to that file. Codebuild was failing to resolve the correct github hash so i just rebased and push a new commit.

The change was meant to be a noop but my idea does the tab to space thing.